CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG](http://keepachangelog.com/) for how to update this file. This project
 adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- Update Elixir version in CI (#390)
```

Comment:
We should be transparent about the new minimum required version (and the fact that the update is because we're tracking Phoenix).

Reply:
I have made changes on that, kindly let me know if it's clear.

Reply:
Much better. I tweaked the language a bit on top of it 👍